modules/canopennode/canopennode.h

```diff
@@ -144,6 +144,31 @@ void canopen_leds_init(CO_NMT_t *nmt,
  */
 void canopen_leds_program_download(bool in_progress);
 
+/**
+ * @brief Callback for incoming CAN message
+ *
+ * This callback will be called from interrupt context and should therefore
+ * return quickly.
+ *
+ * It can be used to e.g. wake the loop polling calling CO_process.
+ */
+typedef void (*canopen_rxmsg_callback)(void);
```

Comment:
Thanks! One last thing that I didn't notice before:

```suggestion
typedef void (*canopen_rxmsg_callback_t)(void);
```